src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    private final Random random = new Random();
+
+    private final int maximumNumber = 100;
+    private final int minimumNumber = 1;
```

Comment:
```suggestion
    private static final int MAXIMUM_NUMBER = 100;
    private static final int MINIMUM_NUMBER = 1;
    
    private final Random random = new Random();
    private final ColorSupplier coloerSupplier = new ColorSupplier()
```

constants should be ```static final ``` and in uppercase